app/profile/controllers.py

```diff
@@ -0,0 +1,7 @@
+import uuid
+from app.profile.repos import UserRepo
+from app.auth import utils
+
+
+def display(user_id):
+    return UserRepo.get_user_by_id(user_id)
```

Comment:
```suggestion
    user = UserRepo.get_user_by_id(user_id)
    return user.to_json()
```
needs to be json serializable